app/deepseg.cc

```diff
@@ -268,15 +326,15 @@ int main(int argc, char* argv[]) try {
 	size_t threads= 2;
 	size_t width  = 640;
 	size_t height = 480;
-	const char *back = nullptr; // "images/background.png";
-	const char *vcam = "/dev/video0";
-	const char *ccam = "/dev/video1";
+	const char *back = nullptr;
+	const char *vcam = "/dev/video1";
+	const char *ccam = "/dev/video0";
 	bool flipHorizontal = false;
```

Comment:
We might want to be more intelligent in defaulting camera paths too (ie: search for them)?